.github/workflows/build_and_test_release_latest.yml

```diff
@@ -0,0 +1,52 @@
+name: Build & Test release-latest
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every hour. This helps detect flakiness,
+    # and broken external dependencies.
+    - cron:  '0 * * * *'
+
+jobs:
+  build_and_test_release_latest:
+    name: Build and Test Release Latest ROS ${{ matrix.ros_version }} ${{ matrix.ros_distro }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distro: [kinetic, melodic]
+        include:
+        - ros_distro: kinetic
+          ubuntu_distro: xenial
+          ros_version: 1
+        - ros_distro: melodic
+          ubuntu_distro: bionic
+          ros_version: 1
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-${{ matrix.ubuntu_distro }}-ros-${{ matrix.ros_distro }}-ros-base-latest
+    steps:
+    # TODO(setup-ros-docker#7): calling chown is necessary for now
+    - run: sudo chown -R rosbuild:rosbuild "$HOME" .
+    # Needed for the CMakeLists.txt setup
+    - run: export ROS_DISTRO=${{ matrix.ros_distro }}
+    # Needed for the CMakeLists.txt setup
+    - run: export ROS_VERSION=${{ matrix.ros_version }}
+    # Needed to access the vcs repos file from the workspace
+    - name: Checkout source
+      uses: actions/checkout@v2
+    - uses: ros-tooling/action-ros-ci@0.0.17
+      with:
+        source-ros-binary-installation: ${{ matrix.ros_distro }}
+        package-name: lex_common_msgs lex_node
+        # schedule runs against the default branch (master), so specify release-latest via repos file
+        vcs-repo-file-url: "${{ github.workspace }}/.github/workflows/release_latest.repos"
+    # lex_common_msgs fails when building tests: Travis config allows tests to be filtered per package
+    - name: Build and run tests
+      run: bash -c 'source /opt/ros/${{ matrix.ros_distro }}/setup.sh && colcon build --packages-select ros_ws/src/lex-ros1/lex_node --cmake-target tests'
```

Comment:
Does this actually run the tests, or just build them? Could you run action-ros-ci multiple times with different arguments instead of invoking colcon directly this way?

Reply:
I'm going to try that (run `actions-ros-ci` multiple times) and see what happens). Otherwise it appears manually attempting to test didn't work as expected and that approach would need to be fixed.

Reply:
[This colcon option](https://github.com/colcon/colcon-cmake/pull/17) seems like the best approach but having local issues with it.